models/brats_mri_generative_diffusion/configs/inference.json

```diff
@@ -2,13 +2,15 @@
     "imports": [
         "$import torch",
         "$from datetime import datetime",
-        "$from pathlib import Path"
+        "$from pathlib import Path",
+        "$from monai.networks import trt_wrap"
     ],
     "bundle_root": ".",
     "model_dir": "$@bundle_root + '/models'",
     "output_dir": "$@bundle_root + '/output'",
     "create_output_dir": "$Path(@output_dir).mkdir(exist_ok=True)",
     "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "trt": "$True",
```

Comment:
If keep the `inference_trt.json`, then don't need this flag anymore.

Thanks.

Reply:
Well yes I did not realize you support merging of the configs - let's keep this discussion in the thread above.